sass/_chatbox.scss

```diff
@@ -121,6 +121,20 @@
             width: var(--mobile-chat-width);
         }
 
+        #service_administration_description {
+            color: #777;
+            background-color:white;
```

Comment:
Looks like you should use `var(--chat-textarea-background-color);` instead of `white;`.